integration_tests/tests/assert_get_query_results_as_dict_objects_equal.sql

```diff
@@ -77,5 +77,5 @@ Instead, we'll manually check that the values of these dictionaries are equivale
 {% endif %}
 
 {{ log(ns.err_msg, info=True) }}
-select 1 {% if ns.pass %} limit 0 {% endif %}
+select 1 as col_name {% if ns.pass %} where 0=1 {% endif %}
```

Comment:
ouch. looks at if TSQL and BQ don't play nice together. From CircleCI

> Query without FROM clause cannot have a WHERE clause

Reply:
Gross. Perhaps we do need a dispatched macro `{{ limit_zero() }}` after all, just for the sake of integration tests

Reply:
where would `limit_zero()` live? as a macro just for integration tests, `integration_tests/macros`? could it be properly dispatched there?

Reply:
Yes, it could live in `integration_tests/macros`, and look like:
```
{% macro limit_zero() %}
    {{ return(adapter.dispatch('limit_zero', dbt_utils._get_utils_namespaces())()) }}
{% endmacro %}

{% macro default__limit_zero() %}
    {{ return('limit 10') }}
{% endmacro %}
```
i.e. It dispatches via the `dbt_utils_dispatch_list` var for namespace purposes, but it doesn't actually "live" in / ship with the `dbt_utils` package.

Then, you should be able to define:
```
{% macro sqlserver__limit_zero() %}
  {{ return('where 0=1') }} 
{% endmacro %}
```

This is the same thing as, or very similar to, what we did with [`prep_external`](https://github.com/fishtown-analytics/dbt-external-tables/blob/master/integration_tests/macros/common/prep_external.sql) over in the external tables package